content/en/getting_started/continuous_testing/_index.md

```diff
@@ -0,0 +1,153 @@
+- [Unused Tests](https://app.datadoghq.com/synthetics/explorer?query=%40ci.job.name%3A%2A&agg_m=count&agg_q=%40result.test_public_id&cols=&index=%2A&top_n=100&track=synbatch&viz=query_table)
+
+{{< img src="continuous_testing/example_search_queries.png" alt="example-search-queries" style="width:100%;" >}}
+
+Optionally, you can create a query to [search your test runs](https://docs.datadoghq.com/continuous_testing/explorer/?tab=testruns#create-a-search-query). Using the browser test you created above, locate the Test ID and create a search query using the common test run facets. To find the ID of your browser test:
+1. Navigate to the Synthetic Tests page
+2. Select a test
+3. Look for the Test ID within the **Properties** section
+
+{{< img src="continuous_testing/example_test_id.png" alt="example_test_id" style="width:70%;" >}}
+
+To export your view of the Synthetics and CT Explorer, click **>Views** and click **Save**. For more information, see [Saved Views](https://docs.datadoghq.com/continuous_testing/explorer/saved_views/).
+
+For more information about using facets in your search query, see [Search Test Runs](https://docs.datadoghq.com/continuous_testing/explorer/search_runs/).
+
+## Set Parallelization preferences
+
+By default, tests are not parallelized. Parallelization allows you to run multiple tests in your CI/CD pipelines simultaneously. If you want to parallelize your tests, you can use the Estimate Parallelization calculator to determine your needs.
+
+Find it by hovering over **UX Monitoring** on the Datadog site, selecting **Settings**, and choosing **Parallelization Settings** on the left to find the calculator in the lower-right.
+
+{{< img src="continuous_testing/parallelization_estimate.png" alt="parallelization_estimate" style="width:100%;" >}}
+
+For instance, if you have 24 tests per CI batch, each taking 2 minutes to complete, and your target is for all tests to be completed within 4 minutes, you’d need to run 12 tests in parallel.
+
+$$\text"estimated parallelization" = {\text"24 tests per CI batch"* \text"2 minute duration"} / \text"4 minute expected duration in your CI pipeline"$$
+
+Once you’re done estimating your parallelization, input the number of test runs you want to execute at the same time in the Parallelization modal. Then click **Save Selection**.
+
+See the [Parallelization documentation](https://docs.datadoghq.com/continuous_testing/settings/#parallelization) for details.
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://www.datadoghq.com/
+[2]: /getting_started/synthetics/api_test/
+[3]: /getting_started/synthetics/api_test/#create-a-multistep-api-test
+[4]: /getting_started/synthetics/browser_test/
+[5]: /getting_started/synthetics/private_location/
```

Comment:
Last bit of feedback is to add the rest of the links in [something]() above down here 🙏🏼